src/App.js

```diff
@@ -1,31 +1,21 @@
-import NotificationPortal from '@redhat-cloud-services/frontend-components-notifications/NotificationPortal';
-import '@redhat-cloud-services/frontend-components-notifications/index.css';
-import { isEqual } from 'lodash';
```

Comment:
If you ever use lodash functions again, please consider the import from the function directory like
```
import isEqual from 'lodash/isEqual';
```
This will let you avoid importing the whole lodash library into your js bundles.

Reply:
You are giving comments about what I got rid of already :)

Reply:
yep, I was rather referring to my previous comment where I suggested to use lodash for deep comparisons. and it implied this comment :-P